kiezcoding.html

```diff
@@ -0,0 +1,147 @@
+      <img alt="BERLIN.JS" src="/img/logo.png" class="logo logo-default" />
+      <hgroup>
+        <h4>Berlin.JS Special Event: Kiezcoding</h4>
+        <h1>Daytime co-working pre-Meetup event</h1>
+      </hgroup>
+    </header>
+    <div role="main" class="content clearfix">
+
+      <section class="announcement">
+
+        <p class="intro"><strong>Kiezcoding</strong> is a unique and fun way to change up your regular work routine. You can have the chance to work alongside other friends (and make new ones!) in the Berlin.JS community!</p>
+
+        <p><strong>Join us at <a href="https://www.google.de/maps/place/co.up/@52.5003298,13.4197864,15z/data=!4m2!3m1!1s0x0:0x2cfd69e5a9f68f1a">co.up</a> on Thursday, April 21</strong> at any time, starting from 10:00.</p>
+
+        <p class="intro">Bring your laptop, your coworkers, and your friends to co.up for a fun, productive day to get your work done outside of your regular workspace! Pair programming and helping each other do better work by solving problems together and sharing your knowledge is highly encouraged, but not enforced.</p>
+        
+        <p>The space has wifi, desk space, and plenty of power outlets for you.</p>
+
+        <p>We'll spend the day working together, and finish up the evening with Berlin’s longest running JavaScript meetup, BerlinJS.</p>
+
+        <p>As always, our events adhere to the <a href="http://berlincodeofconduct.org/">Berlin CoC</a>.</p>
+
+        <p>There’s an <a href="http://www.meetup.com/Berlin-JS/events/jwdcslyvgbcc/">event page on Meetup.com</a> you might join to acknowledge that you’re attending (this is not mandatory).</p>
+
+      </section>
+
+      <section>
+        <div class="two-cols clearfix">
+
+          <div><!-- left -->
+
+            <h2>When is it?</h2>
+            
+            <p>
+              Thursday, April 21<br>
+              10:00 -- Doors open<br>
+              10:15 - 10:30 -- Introductions and Opening Announcements<br>
+              10:30 - 19:00 -- Co-working<br>
+              19:00 - 21:30 -- <a href="http://berlinjs.org">Berlin.JS</a><br>
+            </p>
+            
+            <p>You are also welcome to come at any time</p>
+            
+            <h2>What’s for lunch?</h2>
+              
+            <p>There are a variety of options in the area surrounding Kottbusser Tor. We may also decide to order delivery together, the choice is yours.</p>
+
+          </div>
+
+          <div><!-- right -->
+
+            <h2>How much is it?</h2>
+
+            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, <a href="mailto:WHICH_EMAIL_ADDRESS">email us to make the arrangements</a>.</p>
```

Comment:
I was careful in my choosing of the words “If you are not able to pay the full price for any reason”, so I personally am happy with it. I chose to not mention shame explicitly, but am open to adding something along those lines.

If we want to mention the reason (including everyone in the community) then what about:

“… there will be a need-based sliding scale from 0-15 euro, because we want to welcome everyone to be part of this community, regardless of their current financial status”